snuba/datasets/dataset_schemas.py

```diff
@@ -48,10 +48,3 @@ def get_create_statements(self) -> Sequence[DDLStatement]:
             for schema in self.get_unique_schemas()
             if isinstance(schema, TableSchema)
         ]
-
-    def get_drop_statements(self) -> Sequence[DDLStatement]:
-        return [
-            schema.get_local_drop_table_statement()
```

Comment:
Do we plan to provide a way to delete entirely a dataset (including dropping all the tables)? If not what do we need `get_local_drop_table_statement` for ?

Reply:
Forgot to remove, thanks